integration-fe/src/components/TableInstance/TableInstance.jsx

```diff
@@ -11,35 +13,39 @@ class TableInstance extends Component {
 
   render() {
     let dataRowObj = {};
+
     return (
-      <div className='wrapper table-instance'>
-        <table className='instances'>
-          <thead>
-            <tr>
-              <th><span>Description</span></th>
-              <th><span>Active in</span></th>
-              <th><span>Webhook URL</span></th>
-              <th><span>Last Posted</span></th>
-              <th><span>Actions</span></th>
-            </tr>
-          </thead>
-          <tbody>
-            {
-              this.props.instanceList.map((item, index) => {
-                dataRowObj = {
-                  description: item.description,
-                  appName: this.props.appName,
-                  streamType: item.streamType,
-                  instanceId: item.instanceId,
-                  baseWebhookUrl: this.props.baseWebhookUrl,
-                  postingLocationRooms: item.postingLocationRooms,
-                  lastPosted: item.lastPosted,
-                };
-                return <DataRow instance={dataRowObj} key={index} id={index} />;
-              })
-            }
-          </tbody>
-        </table>
+      <div>
+        <Spinner />
```

Comment:
You should try import the Spinner component inside the Home component, this would avoid you to import the spinner in each child component.